broken-site-reporting/tests.json

```diff
@@ -74,6 +104,102 @@
                     "macos-browser",
                     "windows-browser"
                 ]
+            },
+            {
+                "name": "Test encoding of tracker blocking fields",
+                "siteURL": "https://example.test/",
+                "wasUpgraded": true,
+                "category": "content",
+                "blockedTrackers": ["bad.tracker.test", "tracking.test"],
+                "ignoreRequests": ["ignored1.test", "sub.ignored2.test"],
+                "noActionRequests": ["noaction1.test", "sub.noaction2.test"],
+                "adAttributionRequests": ["adattr1.test", "sub.adattr2.test"],
+                "ignoredByUserRequests": ["ignoreuser1.test", "sub.ignoreuser2.test"],
+                "surrogates": ["surrogate.domain.test", "domain2.com", "nested.sub.domain3.com"],
+                "atb": "v123-456g",
+                "blocklistVersion": "abc123",
+                "expectReportURLPrefix": "https://improving.duckduckgo.com/t/epbf",
+                "expectReportURLParams": [
+                    {"name": "category", "value": "content"},
+                    {"name": "siteUrl", "value": "https%3A%2F%2Fexample.test%2F"},
+                    {"name": "upgradedHttps", "value": "true"},
+                    {"name": "tds", "value": "abc123"},
+                    {"name": "blockedTrackers", "value": "bad.tracker.test,tracking.test"},
+                    {"name": "ignoreRequests", "value": "ignored1.test,sub.ignored2.test"},
+                    {"name": "noActionRequests", "value": "noaction1.test,sub.noaction2.test"},
+                    {"name": "adAttributionRequests", "value": "adattr1.test,sub.adattr2.test"},
+                    {"name": "ignoredByUserRequests", "value": "ignoreuser1.test,sub.ignoreuser2.test"},
+                    {"name": "surrogates", "value": "surrogate.domain.test,domain2.com,nested.sub.domain3.com"}
+                ],
+                "exceptPlatforms": [
+                    "android-browser",
+                    "ios-browser",
+                    "macos-browser",
+                    "safari-extension",
+                    "windows-browser"
+                ]
+            },
+            {
+                "name": "Test truncation when report is too long (URL)",
```

Comment:
nice, is truncation documented? only extension supports it ATM?

Reply:
@kdzwinel I'm adding truncation documentation to the guidelines. It's only extension at the moment